src/database/seeds/0001-insert-base_fields.sql

```diff
@@ -201,7 +201,7 @@ INSERT INTO base_fields (label, description, short_code, data_type, scope) VALUE
 ('Review  Submission ID' , '', 'review_submission_id', 'number', 'proposal' ),
 ('Review Status' , '', 'review_status', 'string', 'proposal' ),
 ('Review Date' , '', 'review_date', 'string', 'proposal' ),
-('Review Submission URL' , '', 'review_submission_url', 'URL', 'proposal' ),
+('Review Submission URL' , '', 'review_submission_url', 'url', 'proposal' ),
```

Comment:
@bickelj I *think* the problem with that approach is that a given seed file would then break / stop working depending on the database state.

For instance, if the DB has been modified to add or remove a given column, then running the "v1" immutable seed file which expects the previous state will result in an error, and so later seed files won't actually be run-able either since there will be no data to modify.

Does that make sense / am I off base?